free-2.md

```diff
@@ -58,12 +102,32 @@ making language design decisions), as opposed to "systems" aspects of the
 project (e.g., implementing a complicated semantics that doesn't require a lot
 of language design)?_
 
+_Most of the work would be on design decision because most of the language has
+already been implemented. So, the basics are there. In recreating the language,
+I would be focused on trying to find different ways to make the language more user
+friendly, hopefully with lots of input from freshman, CS5 students, and Prof
+Dodds._
 
 ### Scope
 _How big an idea is this? How ambitious is this project?_
 
+_This project does not have a huge scope. Most of the documentation discusses
+how they have implemented HMMM and they include all the files necessary to
+learn how they implemented HMMM. So, adding the possible new features and 
+would be the largest part of the project. However, I think that the scope of
+this project can grow as I talk to more people about features that they would
+want to see and see how difficult these implementations are._
 
 ### Benefits and drawbacks
 _Why might this be a good idea for a project? Why might this not be a good idea 
 project?_
 
+_This would be a good idea for a project because it's already implemented in
+Python, a language I am very comfortable with. Additionally, almost all Mudd
+students have seen this language and the freshmen will have just used this
+language right as I am working on this project, so I will have a large pool
+of people to gain feedback from as I am working._  
+_This may not be such a good idea because I'm not certain how possible it is.
+It is quite possible that someone has already tried to do some of the updates
+that I have thought of to HMMM and they were simply not possible. Additionally,
+the scope may be a little small. I worry that there will not be enough to do._
```

Comment:
I agree that there may not be a lot to do in terms of modifying HMMM itself. To a certain extent, HMMM is supposed to reflect how machine code is written, which has limited features and difficult to read.

But I think there could be more work if your project is focuses on making it easier to debug HMMM. The debugging interface is quite limited and could probably be made more user-friendly. As an example, HMMM error messages can probably look better.